CodePushPackage.m

```diff
@@ -458,19 +459,32 @@ + (void)installPackage:(NSDictionary *)updatePackage
         return;
     }
     
-    NSString *previousPackageHash = [self getPreviousPackageHash:error];
-    if (!*error && previousPackageHash && ![previousPackageHash isEqualToString:packageHash]) {
-        NSString *previousPackageFolderPath = [self getPackageFolderPath:previousPackageHash];
-        // Error in deleting old package will not cause the entire operation to fail.
-        NSError *deleteError;
-        [[NSFileManager defaultManager] removeItemAtPath:previousPackageFolderPath
-                                                   error:&deleteError];
-        if (deleteError) {
-            NSLog(@"Error deleting old package: %@", deleteError);
+    if (!removePendingUpdate) {
```

Comment:
I'm confused by this because it looks like the "removePendingUpdate" argument is set to true if there is a pending update, but you're only deleting the current update if there isn't a pending update. Does this need to be changed to "if (removePendingUpdate)" or am I reading it wrong?